broker/spring-zeebe-broker/pom.xml

```diff
@@ -18,7 +18,7 @@
     </dependency>
     <dependency>
       <groupId>io.zeebe</groupId>
-      <artifactId>zeebe-broker-core</artifactId>
+      <artifactId>zeebe-broker</artifactId>
```

Comment:
@spulci was this needed? I can see a problem with names on the artefacts that is quite consistent.. but we should make those changes in a separate PR

Reply:
You're right about one separate PR. I will push a new commit reverting that change while maintaining it locally to compile the project. ;)

Reply:
@spulci I am happy with this change if @menski already accepted the change.. I wonder if you are using the broker integration. In my opinion we should discourage people to build their own brokers. Are you using the Zeebe-broker spring integration? If so, how?

Reply:
Projects implements an annotation for the broekr too `@EnableZeebeBroker`. you can find a test inside io.zeebe.spring.broker.test.BrokerTestApplication class while its definition is inside io.zeebe.spring.broker.EnableZeebeBroker meta interface